core/controllers/classifier_test.py

```diff
@@ -67,9 +65,18 @@ def setUp(self):
             '_c_lw': [],
             '_c_l': []
         }
-        self.job_id = classifier_services.create_classifier_training_job(
-            algorithm_id, interaction_id, self.exp_id, self.exploration.version,
-            'Home', training_data, feconf.TRAINING_JOB_STATUS_PENDING)
+        classifier_training_job = (
+            classifier_services.get_classifier_training_job(
+                self.exp_id, self.exploration.version, 'Home'))
+        self.job_id = classifier_training_job.job_id
+        # Use the mark_training_job_pending function below after this is merged
```

Comment:
@seanlip Nop. This is just a small thingy we can do to optimise this test. I had to update the status manually by using multiple lines whereas it could have been done in a single line in the future. It doesn't make a difference. So, I can make a small PR making that change after Giritheja's PR and this PR is merged. Its not significant enough to merge this PR with develop.